src/utils/routes.js

```diff
@@ -124,6 +130,26 @@ function getSamlAuthLoginCallbackPath(providerKey) {
   return urlUtils.concatParts(samlAuthLoginCallbackPrefix, encodeURIComponent(providerKey));
 }
 
+function isApiPath(path) {
+  return path.startsWith(apiPrefix);
+}
+
+function isResetPasswordPath(path) {
+  return path === resetPasswordPath;
+}
+
+function isLinkExternalAccountPath(path) {
+  return path === linkExternalAccountPath;
```

Comment:
Maybe silly thing, but would `connectExternalAccount` work better? I keep reading 'link' as 'hyperlink', but maybe it's just me 😵‍💫

Reply:
As we call it "connecting" also in the description for the user, it's a good idea to also call it "connect" in code. I've renamed it.

Reply:
👍 nice!